assemblies/features/standard/src/main/feature/feature.xml

```diff
@@ -560,7 +560,7 @@
         <capability>http-service;provider:=felix-httplite</capability>
     </feature>
 
-    <feature name="pax-http-service" description="Pax-Web OSGi HTTP Service">
+    <feature name="pax-http-service" description="Pax-Web OSGi HTTP Service" version="${project.version}">
```

Comment:
Not sure it's a good idea to use `${project.version}`.  I would have used `${pax.web.version}`.